### DISCUSSION TITLE
How identify/filter bot within renovate.json ?

### DISCUSSION THREAD

**Discussion by gkoenig** [renovatebot/renovate#19281] [Request Help] [answered]
Hi,
if there are multiple bots renovating a repo (self-hosted on Gitlab in my case), how to discover/identify the bot within the renovate.json within the target repository ?
e.g.: I want to ignore one bot doing something at all but another bot can do basically everything, how would I configure this in the target repo renovate.json , or what would be the approach in general to cover such a scenario ?

Many thanks in advance

**Comment by rarkins** [comment, marked as answer]
Could you back up a step and describe why you want to run multiple bots in the first place? Maybe we can solve your requirements better there, or at least understand it better

**Reply by gkoenig** [reply]
Many thanks for your prompt feedback !
I was facing a situation that other teams created renovate bots, which (by accident) also renovated my project. Of course, this is a misconfiguration of the other bots.....but on the other hand I was asking myself if it is possible to somehow configure my projects _renovate.json_ in a way that I only consider the renovate-bot I know, and ignoring all other bots. By that I could avoid any weird behaviour of wrongly configured bots patching my project.
Within our Gitlab org there are dozens of teams with hundreds of projects and there is always potential of having a bot going crazy ;) 

I am open to any other approaches how to tackle this.

**Reply by rarkins** [reply]
The only way I can think of for now is that you could set requireConfig=ignored in your own bot's (server side "global") config so that it _ignores_ the contents of any `renovate.json` or other config file in your repos. Then you'd need to configure it entirely with global config and not by repo config

**Reply by gkoenig** [reply]
cool, will have a look into this.
This means I will configure my desired behaviour on the bot "backend" side.....but all other bots , which are touching my project by accident, will consider my local renovate.json. Is it possible to write the renovate.json to _ignore everything (simply doing nothing)_ ? .... then all bots will be skipped, but the "real" bot (which is configured on the backend)
That's my final question on that topic, I promise ;)

**Reply by rarkins** [reply]
Yes you could set enabled=false in your renovate.json to disable all other bots
